packages/webpack/src/runtime/runtime.js

```diff
@@ -48,30 +49,56 @@ const stricterScopeTerminator = freeze(
 /**
  * Enforces the policy for resource imports.
  *
- * @param {string | undefined} requestedResourceId - The ID of the requested
- *   resource.
+ * @template T
```

Comment:
Should this be `@template {object} T`?  Is there any case wherein `T` could be a primitive?

Reply:
T extends something didn't want to work for me at the time of writing this originally. Can fix now.